lib/index.js

```diff
@@ -233,22 +236,18 @@ function getPeerDependencies(current, options) {
   const basePath = options.cwd || './'
   return Object.keys(current).map(pkgName => {
     const path = basePath + 'node_modules/' + pkgName + '/package.json'
+    let peers = {}
     try {
       const pkgData = fs.readFileSync(path)
       const pkg = jph.parse(pkgData)
-      return vm.getCurrentDependencies(pkg, { ...options, dep: 'peer' })
+      peers = vm.getCurrentDependencies(pkg, { ...options, dep: 'peer' })
     }
     catch (e) {
       print(options, 'Could not read peer dependencies for package ' + pkgName + '. Is this package installed?', 'warn')
-      return {}
     }
-  }).reduce((acc, peers) => {
-    Object.entries(peers).forEach(([pkgName, version]) => {
-      if (acc[pkgName] === undefined) {
-        acc[pkgName] = []
-      }
-      acc[pkgName][acc[pkgName].length] = version
-    })
+    return [pkgName, peers]
+  }).reduce((acc, [pkgName, peers]) => {
+    acc[pkgName] = peers
     return acc
   }, {})
```

Comment:
Here you can use [_.fromPairs](https://lodash.com/docs/4.17.15#fromPairs), or, even better, you can convert the above `map` into a `reduce` and avoid the additional loop.